modules/coreos-layering-configuring-on.adoc

```diff
@@ -184,7 +162,7 @@ spec:
   desiredConfig:
     name: rendered-layered-ad5a3cad36303c363cf458ab0524e7c0
   machineOSConfig:
-    name: layered
+    name: layered <1>
```

Comment:
There are two <1> labels in this resource. There is one <1> label in the "name" field and another <1> label in the "renderedImagePushspec" field